src/mongofinil/core.clj

```diff
@@ -74,24 +74,45 @@
             (apply f @(first args) (rest args)))
    :else f))
 
+(defn only-true-keys
+  "Construct a set from opts given to :only.
+
+`only-opts` is either a collection or a map where all values are true (excluding :_id).
```

Comment:
Thanks for the review @marcomorain !

> Can't some values be false, to indicate the desire to not return the field?

https://docs.mongodb.org/manual/tutorial/project-fields-from-query-results/

>> You cannot combine inclusion and exclusion semantics in a single projection with the exception of the _id field.